libraries/AP_RangeFinder/AP_RangeFinder_Params.cpp

```diff
@@ -0,0 +1,139 @@
+#include <AP_Common/AP_Common.h>
+#include <AP_Vehicle/AP_Vehicle_Type.h>
+#include "AP_RangeFinder_Params.h"
+#include "AP_RangeFinder.h"
+
+// table of user settable parameters
+const AP_Param::GroupInfo AP_RangeFinder_Params::var_info[] = {
+    // @Param: TYPE
+    // @DisplayName: Rangefinder type
+    // @Description: What type of rangefinder device that is connected
+	    // @Values: 0:None,1:Analog,2:MaxbotixI2C,3:LidarLiteV2-I2C,5:PX4-PWM,6:BBB-PRU,7:LightWareI2C,8:LightWareSerial,9:Bebop,10:MAVLink,11:uLanding,12:LeddarOne,13:MaxbotixSerial,14:TeraRangerI2C,15:LidarLiteV3-I2C,16:VL53L0X,17:NMEA,18:WASP-LRF,19:BenewakeTF02,20:BenewakeTFmini
+    // @User: Standard
+    AP_GROUPINFO("TYPE",    0, AP_RangeFinder_Params, type, 0),
```

Comment:
please don't use index 0 in new parameter tables (there is an issue with zero)

Reply:
Fixed.